src/features/home/components/modules/VideoModule.tsx

```diff
@@ -135,4 +137,6 @@ const VideoTitle = styled(Typo.Title4)(({ theme }) => ({
   textAlign: 'left',
 }))
 
-const StyledTouchable = styledButton(Touchable)``
+const StyledTouchableHighlight = styled.TouchableHighlight(({ theme }) => ({
+  borderRadius: theme.borderRadius.radius,
+}))
```

Comment:
Ca permet d'éviter d'importer `theme` pour `underlayColor`
```suggestion
const StyledTouchableHighlight = styled.TouchableHighlight.attrs(({ theme }) => ({
  underlayColor: theme.colors.white,
}))(({ theme }) => ({
  borderRadius: theme.borderRadius.radius,
}))
```